app/controllers/users_controller.rb

```diff
@@ -3,4 +3,11 @@ def show
     @user = User.find_by!(username: params.fetch(:username))
   end
 
```

Comment:
Hi Young, you seem to be missing the liked and feed actions in this controller. Can you please add these.